### ISSUE TITLE
Add explicit width and height for image elements.

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-hugo-themes#650]

**Comment by MAbdurrehman12** [comment]
@pdpinch yes, we can achieve this via Hugo image processing and in order to process an image we must access the image as either a page resource or a global resource (as stated [here](https://gohugo.io/content-management/image-processing/#image-resources)). 
Images can be accessed as global resources if they are placed in `assets/` directory but our images (of `base-theme` and `www theme`) are placed in `static` directory, outside `assets` so I'm not sure whether we will be able to process these images via Hugo
